guide/additional-features/cooldowns.md

```diff
@@ -71,7 +71,7 @@ if (timestamps.has(interaction.user.id)) {
 
 	if (now < expirationTime) {
 		const expiredTimestamp = Math.round(expirationTime / 1000);
-		return interaction.reply({ content: `Please wait <t:${expiredTimestamp}:R> more second(s) before reusing the \`${command.data.name}\` command.`, ephemeral: true });
+		return interaction.reply({ content: `Please wait, you are on a cooldown for \`${command.data.name}\`. You can use it again <t:${expiredTimestamp}:R>`, ephemeral: true });
```

Comment:
maybe use [time](https://old.discordjs.dev/#/docs/discord.js/main/function/time) function

Reply:
🤷 I mean I could, but I see no need to.